### PR TITLE
ADD: Team Card Component

### LINKED ISSUE #8: Team Card Section

### PR DESCRIPTION
**Pull request by BIPS8812**
#8

### CODE REVIEW

File: src/Components/TeamCard/TeamCard.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  return (
+    <div className={style.cardContainer}>
+      <div className={style.cardImage}>
+        <img src="/images/tom.png" alt="Member" />
+      </div>
+      <div className={style.cardBody}>
+        <div className={style.mediaLinks}>
+          <a
+            aria-label="Social Media"
+            href="https://www.facebook.com/ronny.kumar.7186896"
```
Comment (codadept) [inline src/Components/TeamCard/TeamCard.jsx]: add target="_blank" to all the <a> tags to open the link in a new tab

File: src/Components/TeamCard/TeamCard.module.scss [hunk 2]
```diff
@@ -0,0 +1,83 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap");
```
Comment (codadept) [inline src/Components/TeamCard/TeamCard.module.scss]: Poppins is already imported in the index.scss page just use as

```css
font-family: var(--srijan4-font-1);
```

**Review by codadept — CHANGES_REQUESTED** [review]

**Comment by codadept** [comment]
According to me there should be a bit of box shadow on the team card to show that it can be hovered. Just have a talk with Maharnav and figure it out?

File: src/Components/TeamCard/TeamCard.jsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Icon } from "@iconify/react";
+import style from "./TeamCard.module.scss";
+
+const TeamCard = () => {
+  return (
+    <div className={style.cardContainer}>
+      <div className={style.cardImage}>
+        <img src="/images/tom.png" alt="Member" />
+      </div>
```
Comment (JOSU9435) [inline src/Components/TeamCard/TeamCard.jsx]: pass props down the component for the image

File: src/Components/TeamCard/TeamCard.jsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+          <a
+            aria-label="Social Media"
+            href="https://www.facebook.com/ronny.kumar.7186896"
+          >
+            <Icon className={style.Links} icon="ri:facebook-circle-fill" />
+          </a>
+          <a
+            aria-label="Social Media"
+            href="https://www.linkedin.com/in/bipangshu-saha-2aab9b228/"
+          >
```
Comment (JOSU9435) [inline src/Components/TeamCard/TeamCard.jsx]: - pass props down the component for social links
- open social links in a new tab using `taget="_blank"` and `rel="noreferrer"`

File: src/Components/TeamCard/TeamCard.jsx [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+            <Icon className={style.Links} icon="mdi:linkedin" />
+          </a>
+        </div>
+        <div className={style.memberName}>Bipangshu Saha</div>
+        <div className={style.memberDesignation}>Developer</div>
```
Comment (JOSU9435) [inline src/Components/TeamCard/TeamCard.jsx]: pass props down the component for name and position

File: src/Components/TeamCard/TeamCard.module.scss [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+.cardContainer {
+  position: relative;
+  width: 12rem;
+  height: 16rem;
+  &:hover {
+    .cardImage > img {
+      transform: scale(0.7) translate(-10%, -5%);
+      box-shadow: 0px 4px 100px rgba(6, 0, 0, 0.66);
```
Comment (JOSU9435) [inline src/Components/TeamCard/TeamCard.module.scss]: refrain from using pixels instead use relative units

File: src/Components/TeamCard/TeamCard.module.scss [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+    transition: all 0.3s ease-in-out;
+  }
+
+  .cardBody {
+    transform: translateY(25%);
+    opacity: 0;
+    width: 100%;
+    height: 100%;
+    @include flex-dispay(column, center, start);
+    background: linear-gradient(180deg, #cd6880 28.13%, rgba(9, 145, 222, 0.38) 100%);
```
Comment (JOSU9435) [inline src/Components/TeamCard/TeamCard.module.scss]: use CSS variables for gradients, colors, etc.

File: src/Components/TeamCard/TeamCard.module.scss [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+    .memberName {
+      padding: 0 3%;
+      font-family: "Poppins";
+      font-style: normal;
+      font-weight: 500;
+      font-size: 1.2rem;
+      line-height: 2rem;
+
+      color: #ffffff;
```
Comment (JOSU9435) [inline src/Components/TeamCard/TeamCard.module.scss]: use CSS variables for gradients, colors etc.

File: src/Components/TeamCard/TeamCard.module.scss [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+    }
+
+    .mediaLinks {
+      @include flex-dispay(column, space-around, end);
+      height: 8rem;
+      width: 100%;
+      padding-right: 3%;
+
+      .Links {
+        color: #fff;
```
Comment (JOSU9435) [inline src/Components/TeamCard/TeamCard.module.scss]: use CSS variables for gradients, colors etc.

**Review by JOSU9435 — CHANGES_REQUESTED** [review]

**Comment by codadept** [comment]
Instead of using @iconify use react-icons, as it is used by others

**Review by codadept — APPROVED** [review]

**Pull request merged**